src/EnergyPlus/ZoneContaminantPredictorCorrector.cc

```diff
@@ -2606,6 +2631,143 @@ namespace ZoneContaminantPredictorCorrector {
 
                 ZoneAirCO2(ZoneNum) = ZoneAirCO2Temp(ZoneNum);
 
+                // Hybrid modeling with CO2 concentration starts here.
+                GetHybridModelZone();
```

Comment:
Why add this here? `GetHybridModelZone()` is already called in `ZoneTempPredictorCorrector::GetZoneAirSetPoints`  and there is no logic in `GetHybridModelZone()`  to prevent it from processing the input twice.  Seems like the call from `ZoneTempPredictorCorrector::GetZoneAirSetPoints` should be adequate.

Reply:
The GetHybridModelZone() was moved to ZoneTempPredictorCorrector::GetZoneAirSetPoints in the latest commit.